files/usr/bin/configure-ssh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+[ -z "$SSH_PRIVATE_KEY" ] && echo "Need to set SSH_PRIVATE_KEY" && exit 1;
+[ -z "$DEPLOY_HOST" ] && echo "Need to set DEPLOY_HOST" && exit 1;
+[ -z "$DEPLOY_HOST_KEY" ] && echo "Need to set DEPLOY_HOST_KEY" && exit 1;
+
+# Install SSH public key
+mkdir -p ~/.ssh
```

Comment:
The `~/.ssh` directory is usually chmodded with `0700`.

I know, it also works without it, but it is more “correct” :wink: